README.md

```diff
@@ -368,13 +368,15 @@ Use the `account` subcommand to delete a specific account for the passed account
 The following flags are available for `delete account`:
 
 ```
---dry-run                    If false, deletes the account for the given account ID, defaults to true.
--h, --help                   help for delete
+--dry-run                       If false, deletes the account for the given account ID, defaults to true.
+--delete-associated-resources   If true, deletes the associated resources like registry credentials, role bindings, etc. along with the account.
```

Comment:
It would be good to list all the resources that are affected by this. Do we know/ can we figure this information?

Reply:
Yes. Added all of them in readme

Reply:
thanks!